open_fdd/gateway/local_codex_cli.py

```
@@ -0,0 +1,483 @@
+        p = Path(override)
+        if p.is_file():
+            return str(p.resolve())
+
+    w = shutil.which("codex")
+    if w:
+        return w
+
+    if os.name == "nt":
+        for arg in ("codex.cmd", "codex"):
+            try:
+                r = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)